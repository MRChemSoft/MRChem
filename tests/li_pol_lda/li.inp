# vim:syntax=sh:

world_prec = 1.0e-3               # Overall relative precision
world_size = 5                    # Size of simulation box 2^n

MPI {
  numerically_exact = true        # Guarantee identical results in MPI
}

Molecule {
  multiplicity = 2
$coords
Li      0.0000  0.0000  0.0000
$end
}

WaveFunction {
  method = LDA                    # Wave function method (HF or DFT)
  restricted = false
}

Properties {
  polarizability = true
}

SCF {
  run = false
  guess_type = SAD_DZ
}

Response {
  kain = 3                        # Length of KAIN iterative history
  max_iter = 10
  orbital_thrs = 1.0e-2           # Convergence threshold in orbital residual
  run = [false,false,true]
}

Polarizability {
  frequency = [0.0]
}
