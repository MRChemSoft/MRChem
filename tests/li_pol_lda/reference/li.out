                                                                      
                                                                      
**********************************************************************
***                                                                ***
***                                                                ***
***        __  __ ____   ____ _                                    ***
***       |  \/  |  _ \ / ___| |__   ___ _ __ ___                  ***
***       | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                 ***
***       | |  | |  _ <| |___| | | |  __/ | | | | |                ***
***       |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                ***
***                                                                ***
***       VERSION            0.3.0-alpha                           ***
***                                                                ***
***       Git branch         rebase/mw-guess                       ***
***       Git commit hash    f4882d276365e07f1ad7                  ***
***       Git commit author  Stig Rune Jensen                      ***
***       Git commit date    Sat Mar 7 13:02:21 2020 +0100         ***
***                                                                ***
***       Magnar Bjorgve     <magnar.bjorgve@uit.no>               ***
***       Roberto Di Remigio <roberto.di.remigio@uit.no>           ***
***       Luca Frediani      <luca.frediani@uit.no>                ***
***       Gabriel Gerez      <gsa017@post.uit.no>                  ***
***       Stig Rune Jensen   <stig.r.jensen@uit.no>                ***
***       Peter Wind         <peter.wind@uit.no>                   ***
***                                                                ***
**********************************************************************

----------------------------------------------------------------------

 MPI processes         :      (no bank)                             1
 OpenMP threads        :                                            1
 Total cores           :                                            1
                                                                      
----------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

----------------------------------------------------------------------

 MRCPP version         : 1.2.0-alpha3
 Git branch            : rebase/mw-guess
 Git commit hash       : f4882d276365e07f1ad7
 Git commit author     : Stig Rune Jensen
 Git commit date       : Sat Mar 7 13:02:21 2020 +0100

 Linear algebra        : EIGEN v3.3.7
 Parallelization       : OpenMP

----------------------------------------------------------------------


                                                                      
======================================================================
                       MultiResolution Analysis
----------------------------------------------------------------------
 polynomial order      : 5
 polynomial type       : Interpolating
----------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   16.00000    16.00000    16.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -16.00000   -16.00000   -16.00000 ]
 upper bounds          : [   16.00000    16.00000    16.00000 ]
 total length          : [   32.00000    32.00000    32.00000 ]
======================================================================


                                                                      
**********************************************************************
***                                                                ***
***                     Initializing Molecule                      ***
***                                                                ***
**********************************************************************
                                                                      
                                                                      
======================================================================
                               Molecule
----------------------------------------------------------------------
 Charge                :                                            0
 Multiplicity          :                                            2
----------------------------------------------------------------------
    N    Atom          :              x              y              z
----------------------------------------------------------------------
    0      Li          :       0.000000       0.000000       0.000000
----------------------------------------------------------------------
 Center of mass        :       0.000000       0.000000       0.000000
 Gauge origin          :       0.000000       0.000000       0.000000
======================================================================


                                                                      
**********************************************************************
***                                                                ***
***              Computing Ground State Wavefunction               ***
***                                                                ***
**********************************************************************
                                                                      
                                                                      
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation           : Diagonalize Hamiltonian matrix
 Precision             : 1.00000e-03
 Restricted            : False
 Hamiltonian           : Superposition of Atomic Densities (SAD)
 Functional            : LDA (SVWN5)
 AO basis              : Hydrogenic orbitals
 Zeta quality          : 2
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


======================================================================
                          Molecular Orbitals
----------------------------------------------------------------------
 Alpha electrons       :                                            2
 Beta electrons        :                                            1
 Total electrons       :                                            3
----------------------------------------------------------------------
    n  Occ Spin        :                                         Norm
----------------------------------------------------------------------
    0    1    a        :                           1.000000000000e+00
    1    1    a        :                           9.999999447285e-01
    2    1    b        :                           1.000000000000e+00
======================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation           : Compute initial energy
 Method                : DFT (SVWN5)
 Precision             : 1.00000e-03
 Localization          : Off
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


======================================================================
                           Molecular Energy
----------------------------------------------------------------------
 Kinetic energy        :           (au)                7.933164943367
 E-N energy            :           (au)              -17.614352857843
 Coulomb energy        :           (au)                4.103869443011
 Exchange energy       :           (au)                0.000000000000
 X-C energy            :           (au)               -1.719708897512
 Ext. field (el)       :           (au)                0.000000000000
 Ext. field (nuc)      :           (au)                0.000000000000
----------------------------------------------------------------------
 Electronic energy     :           (au)               -7.297027368978
 Nuclear energy        :           (au)                0.000000000000
----------------------------------------------------------------------
 Total energy          :           (au)           -7.297027368978e+00
                       :     (kcal/mol)           -4.578953769586e+03
                       :       (kJ/mol)           -1.915834258438e+04
                       :           (eV)           -1.985622214576e+02
======================================================================


======================================================================
                           Orbital Energies
----------------------------------------------------------------------
    n  Occ Spin        :                                      Epsilon
----------------------------------------------------------------------
    0    1    a        :           (au)               -1.814574765685
    1    1    a        :           (au)               -0.114315988999
    2    1    b        :           (au)               -1.805755227476
----------------------------------------------------------------------
 Sum occupied          :           (au)               -3.734645982160
======================================================================


                                                                      
**********************************************************************
***                                                                ***
***             Computing Linear Response Wavefunction             ***
***                                                                ***
**********************************************************************
                                                                      
                                                                      
======================================================================
                          Molecular Orbitals
----------------------------------------------------------------------
 Alpha electrons       :                                            2
 Beta electrons        :                                            1
 Total electrons       :                                            3
----------------------------------------------------------------------
    n  Occ Spin        :                                         Norm
----------------------------------------------------------------------
    0    1    a        :                          -1.000000000000e+00
    1    1    a        :                          -1.000000000000e+00
    2    1    b        :                          -1.000000000000e+00
======================================================================


======================================================================
                          Molecular Orbitals
----------------------------------------------------------------------
 Alpha electrons       :                                            2
 Beta electrons        :                                            1
 Total electrons       :                                            3
----------------------------------------------------------------------
    n  Occ Spin        :                                         Norm
----------------------------------------------------------------------
    0    1    a        :                          -1.000000000000e+00
    1    1    a        :                          -1.000000000000e+00
    2    1    b        :                          -1.000000000000e+00
======================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation           : Static linear response
 Method                : DFT (SVWN5)
 Checkpointing         : Off
 Perturbation          : h_E_dip[z]
 Max iterations        : 10
 KAIN solver           : 3
 Start precision       : 1.00000e-03
 Final precision       : 1.00000e-03
 Helmholtz precision   : 1.00000e-03
 Property threshold    : Off
 Orbital threshold     : 1.00000e-02
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


======================================================================
 Iter         MO residual      Symmetric property              Update
----------------------------------------------------------------------
    0        1.000000e+00          0.000000000000        0.000000e+00
    1        1.185033e+01        -54.526882534344       -5.452688e+01
    2        6.333495e+00       -117.270492244169       -6.274361e+01
    3        1.553630e+00       -121.463054522036       -4.192562e+00
    4        1.193994e+00       -137.384300758716       -1.592125e+01
    5        3.497740e-01       -138.782166142373       -1.397865e+00
    6        1.631738e-01       -140.088218290776       -1.306052e+00
    7        6.046424e-02       -140.974359759514       -8.861415e-01
    8        1.313929e-02       -141.053953987417       -7.959423e-02
    9        3.779161e-03       -141.061414022408       -7.460035e-03
----------------------------------------------------------------------
                   SCF converged in 9 iterations!
======================================================================


                                                                      
**********************************************************************
***                                                                ***
***                 Printing Molecular Properties                  ***
***                                                                ***
**********************************************************************
                                                                      
                                                                      
======================================================================
                               Molecule
----------------------------------------------------------------------
 Charge                :                                            0
 Multiplicity          :                                            2
----------------------------------------------------------------------
    N    Atom          :              x              y              z
----------------------------------------------------------------------
    0      Li          :       0.000000       0.000000       0.000000
----------------------------------------------------------------------
 Center of mass        :       0.000000       0.000000       0.000000
 Gauge origin          :       0.000000       0.000000       0.000000
======================================================================


======================================================================
                           Orbital Energies
----------------------------------------------------------------------
    n  Occ Spin        :                                      Epsilon
----------------------------------------------------------------------
    0    1    a        :           (au)               -1.814574765685
    1    1    a        :           (au)               -0.114315989000
    2    1    b        :           (au)               -1.805755227476
----------------------------------------------------------------------
 Sum occupied          :           (au)               -3.734645982161
======================================================================


======================================================================
                           Molecular Energy
----------------------------------------------------------------------
 Kinetic energy        :           (au)                7.933164943367
 E-N energy            :           (au)              -17.614352857843
 Coulomb energy        :           (au)                4.103869443011
 Exchange energy       :           (au)                0.000000000000
 X-C energy            :           (au)               -1.719708897512
 Ext. field (el)       :           (au)                0.000000000000
 Ext. field (nuc)      :           (au)                0.000000000000
----------------------------------------------------------------------
 Electronic energy     :           (au)               -7.297027368978
 Nuclear energy        :           (au)                0.000000000000
----------------------------------------------------------------------
 Total energy          :           (au)           -7.297027368978e+00
                       :     (kcal/mol)           -4.578953769586e+03
                       :       (kJ/mol)           -1.915834258438e+04
                       :           (eV)           -1.985622214576e+02
======================================================================


======================================================================
                            Polarizability
----------------------------------------------------------------------
 Frequency             :           (au)                      0.000000
                       :         (cm-1)                      0.000000
----------------------------------------------------------------------
 Total tensor          :            N/A            N/A            N/A
                       :            N/A            N/A            N/A
                       :      -0.000000      -0.000000     141.061414
 Isotropic average     :           (au)                           N/A
======================================================================


                                                                      
                                                                      
**********************************************************************
***                                                                ***
***                         Exiting MRChem                         ***
***                                                                ***
***                    Wall time :  0h  1m 33s                     ***
***                                                                ***
**********************************************************************
                                                                      
                                                                      
